opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -245,34 +244,39 @@ impl<RT: Runtime> PeriodicReaderWorker<RT> {
             Either::Left((res, _)) => {
                 res // return the status of export.
             }
-            Either::Right(_) => {
-                otel_error!(
-                    name: "collect_and_export",
-                    status = "timed_out"
-                );
-                Err(MetricsError::Other("export timed out".into()))
-            }
+            Either::Right(_) => Err(MetricsError::Other("export timed out".into())),
         }
     }
 
     async fn process_message(&mut self, message: Message) -> bool {
         match message {
             Message::Export => {
                 if let Err(err) = self.collect_and_export().await {
-                    global::handle_error(err)
+                    otel_error!(
+                        name: "PeriodicReader.ExportFailed",
+                        message = "Failed to export metrics",
+                        reason = format!("{}", err));
                 }
             }
             Message::Flush(ch) => {
                 let res = self.collect_and_export().await;
-                if ch.send(res).is_err() {
-                    global::handle_error(MetricsError::Other("flush channel closed".into()))
+                if let Err(send_error) = ch.send(res) {
+                    otel_debug!(
+                        name: "PeriodicReader.Flush.SendResultError",
+                        message = "Failed to send flush result",
+                        reason = format!("{:?}", send_error),
+                    );
```

Comment:
This doesn't log actual flush error, but failure to send the result status to the channel. So keeping it debug